srcpkgs/passff-host/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'passff-host'
+pkgname=passff-host
+version=1.2.1
+revision=1
+create_wrksrc=yes
+depends="python3"
+short_desc="Host app for the WebExtension PassFF"
+maintainer="Colin Reeder <colin@vpzom.click>"
+license="GPL-2.0-or-later"
```

Comment:
-only

Reply:
I wasn't sure which one to choose, how is that determined?

Reply:
Citing GPL, 

> Each version is given a distinguishing version number.  If the Program
specifies a version number of this License which applies to it and "any
later version", you have the option of following the terms and conditions
either of that version or of any later version published by the Free
Software Foundation.  If the Program does not specify a version number of
this License, you may choose any version ever published by the Free Software
Foundation.